ktor-http/common/src/io/ktor/http/Mimes.kt

```diff
@@ -6,7 +6,7 @@ package io.ktor.http
 
 import io.ktor.util.*
 
-internal const val INITIAL_MIMES_LIST_SIZE: Int = 1217
+internal const val INITIAL_MIMES_LIST_SIZE: Int = 1216
```

Comment:
Updated this as `audio/mpeg,mpg` has been removed, it duplicates 

https://github.com/ktorio/ktor/blob/73f83874c756a5441a04984ca48e505a37027b58/ktor-http/common/src/io/ktor/http/Mimes.kt#L706